app/models/verification.js

```diff
@@ -0,0 +1,36 @@
+
+module.exports = function(sequelize, Sequelize) {
```

Comment:
Email Verification is applicable for Clients only. So, can we add a Foreign Key Constraint for the Verification Model?

Reply:
Yes, but for vendor verification we would still need a factor to verify them once it gets verified offline, so at that time we can use this same module to verify the vendors also

Reply:
We can add the mailing option only for clients, that can be done, but I feel we would still need the verification module for vendors.